monai/apps/datasets.py

```diff
@@ -306,41 +312,33 @@ def _split_datalist(self, datalist: List[Dict]) -> List[Dict]:
             return [datalist[i] for i in self.indices]
 
 
-class CVDecathlonDataset:
+class CrossValidation:
```

Comment:
this looks nice, might need another PR to unify the `apps.dataset` interfaces to support CV.

Reply:
Yes, if we can unify all the dataset APIs, that would be more clear.
Thanks.